### PR TITLE
Prettify

### PR DESCRIPTION
**Pull request by Marie-Garde**

### CODE REVIEW

**Review by Thoms-C — DISMISSED** [review]
Looks ok to me

**Review by LoicBrassart — DISMISSED** [review]
SnE

**Review by Gauthier-Lhomme — APPROVED** [review]

File: index.js [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+const express = require("express");
+const app = express();
+const cors = require("cors");
+
+app.use(cors());
+app.use(express.json());
+
+app.get("/", (req, res) => {
+  res.send("Hi there!");
+});
```
Comment (LoicBrassart) [inline index.js:11]: Useless route

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**